src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -210,4 +210,4 @@ Temporarily using the Windows Terminal default settings.
   <data name="CloseWindowWarningTitle" xml:space="preserve">
     <value>Do you want to close all tabs?</value>
   </data>
-</root>
+</root>
```

Comment:
Same here. Revert changes to this file.

Reply:
Not sure what you want me to do here. Everywhere I've looked there doesn't appear to be anything to revert.